source/read-write-concern/tests/operation/default-write-concern-2.6.json

```diff
@@ -1,19 +1,55 @@
 {
```

Comment:
Nothing major, just noted that for other tests you created a `unified` directory under tests, whereas this PR keeps original files.

Reply:
This was intentional, as the tests were already under an `operation/` directory. In other specs, legacy tests were either in `tests/` or, in the case of CRUD, a specific directory.

These tests are old and few enough that I'm not worried at all about people mixing up legacy and unified test files.